### PR TITLE
test: WebSocket ping

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/api-client/src/main/tcp/ReconnectingWebsocket.test.node.js [hunk 1]
```diff
@@ -50,22 +63,26 @@ function startEchoServer() {
...
+
+  beforeEach(() => {
+    this.server = startEchoServer();
+  });
 
   afterEach(done => {
-    if (server) {
-      server.close(() => {
-        server = undefined;
+    if (this.server) {
```
Comment (ffflorian) [inline packages/api-client/src/main/tcp/ReconnectingWebsocket.test.node.js]: Let's keep the global variable.

**Comment by Yserz** [comment]
It doesn't connect to a live server. I just missed a `this`.

File: packages/api-client/src/main/tcp/ReconnectingWebsocket.test.node.js [hunk 2]
```diff
@@ -19,15 +19,28 @@
...
-const WEBSOCKET_URL = `ws://127.0.0.1:${WEBSOCKET_PORT}`;
-let server = undefined;
+const reservedPorts = [];
 
 function startEchoServer() {
-  server = new WebSocketServer({port: WEBSOCKET_PORT});
+  const getWebsocketPort = () => {
+    const WEBSOCKET_START_PORT = 8087;
+    let currentPort = WEBSOCKET_START_PORT;
+    while (true) {
```
Comment (ffflorian) [inline packages/api-client/src/main/tcp/ReconnectingWebsocket.test.node.js]: Can't we also do `while(currentPort++)` here?

Comment (ffflorian) [reply packages/api-client/src/main/tcp/ReconnectingWebsocket.test.node.js]: https://github.com/wireapp/wire-web-packages/pull/2286/files

**Pull request merged**